.gitignore

```diff
@@ -10,7 +10,7 @@ node_modules/
 
 yarn-error.log
 
-.deps/tmp/
+.deps/problems.md
```

Comment:
I believe we should not ignore this file.

Reply:
I have fixed it.